src/display/editor/freetext.js

```diff
@@ -550,6 +550,8 @@ class FreeTextEditor extends AnnotationEditor {
       return this.div;
     }
 
+    this._uiManager._editorUndoBar?.hide();
```

Comment:
Here and in the other types of editors, isn't it possible to get rid of this ?

Reply:
All editors except "ink" call this in the `render` method. If you're comfortable with me removing that special case, I could move this to `AnnotationEditor#render`.

Reply:
@calixteman done with the caveat mentioned above (in case of the ink editor, the bar is dismissed when it's selected from the toolbar and not when it's actually created). Check it out and let me know what you think.

Reply:
Unmarking this comment as unresolved so you could consider this before the final go-ahead.

Reply:
To be clear, now if you:
- draw an ink annotation
- delete it
- draw another one

the popup bar does not close.

Reply:
This is fixed by moving the hide action to the parent class while keeping the special case for ink.js, thanks for the pointer @calixteman